src/components/App.js

```diff
@@ -0,0 +1,10 @@
+import { Provider } from 'react-redux';
+import store from '../Redux/store/index';
+
+
+const App = ({ store }) => (
+  <Provider store={store}>
+
+  </Provider>
+)
+ReactDOM.render(<App store={store} />, document.getElementById('app'));
```

Comment:
I think it will be more clearer if global store is configured separately in `App.js`. Then import it in `index.js` file to access `dom`.

``` js
// App.js 

import store from './redux';

const App = () => {
  <provider store={store}>
      <App />   // we will add routes and components so that we can access data.
  </provider>
};

export default App;

```


``` js
// Index.js

import App from './App';

ReactDOM.render(<App />, document.getElementById('app'));
```